lib/ansible/plugins/action/reboot.py

```diff
@@ -246,8 +246,8 @@ def validate_reboot(self):
 
         try:
             # keep on checking system boot_time with short connection responses
-            reboot_timeout = int(self._task.args.get('reboot_timeout', self._task.args.get('reboot_timeout_sec', self.DEFAULT_REBOOT_TIMEOUT)))
-            connect_timeout = self._task.args.get('connect_timeout', self._task.args.get('connect_timeout_sec', self.DEFAULT_CONNECT_TIMEOUT))
+            reboot_timeout = int(self._task.args.get('reboot_timeout', self.DEFAULT_REBOOT_TIMEOUT))
```

Comment:
Both `reboot_timeout_sec` and `connect_timeout_sec` are valid for `win_reboot`. Since `win_reboot` is using this method, these options need to stay.

Reply:
You are right, I thought we were overloading this, but we aren't.